packages/modules/src/modules/banners/contributions/ContributionsBannerSecondaryCta.tsx

```diff
@@ -14,6 +14,11 @@ import { hasSetReminder } from '../../utils/reminders';
 const reminderButtonStyles = css`
     color: ${neutral[0]};
     margin-left: ${space[4]}px;
+
+    /* TODO: Remove this after Giving Tuesday */
+    @media (min-width: 740px) and (max-width: 769px) {
+        display: none;
+    }
```

Comment:
We're hiding it at this breakpoint as the cta overflows :( I've added a screenshot to demonstrate

Reply:
Just interested is hiding the reminder preferable to reducing the `font-size` at this breakpoint?

Reply:
haha that's a good point! I'll merge this now, but I'll also ping ellen and see if she's happy with the design with the small font size